Update/kelola akun

bayun2445 opened the pull request:
added button edit and delete
added new functions in RegisterConroller
created edit-akun page

bayun2445 committed:
added edit and delete feature

rinaltair commented:
feature edit + delete for kelola akun in admin -> approve

rinalditoryanari commented:
Logbook
1. kalo udah di save, file nya tidak muncul di sisi mhsw (baik tampilan detail atau edit)
   - dospen nya bisa lihat, tapi mhsw nya gak bisa
   - pi nya gak bisa liat

Laporan + Sertif
2. dokumen sama sertif gak bisa dibuka samsek

rinaltair reviewed (changes requested):
Logbook
1. kalo udah di save, file nya tidak muncul di sisi mhsw (baik tampilan detail atau edit)
   - dospen nya bisa lihat, tapi mhsw nya gak bisa
   - pi nya gak bisa liat

Laporan + Sertif
1. dokumen sama sertif gak bisa dibuka samsek

bayun2445 committed:
Merge remote-tracking branch 'upstream/master' into update/kelola-akun

lanang0809-pro committed:
Update Tampilan Halaman Logbook PI

lanang0809-pro committed:
Merge branch 'update/kelola-akun' of https://github.com/bayun2445/SIMBKM into update/kelola-akun

bayun2445 committed:
Moved view dokumen logbook

Moved view dokumen logbook button from edit page to detail page.

bayun2445 commented:
- Added view logbook feeature for Pembimbing Industri
- Moved view dokumen logbook for Mahasiswa from edit to detail page 
- Added update dokumen form in edit logbook page

rinaltair commented:
done

Chillrend commented:
lgtm
